zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -5773,6 +5773,9 @@ components:
         name:
           description: Name of this process definition.
           type: string
+        isLatestVersion:
+          description: Whether to only return the latest version of each process definition. When using this filter, paging backwards is not supported with Elasticsearch or Opensearch.
```

🔧 I think we need to be more specific about what "paging backwards" means. Please double-check if my suggestions are actually correct.
```suggestion
          description: >
            Whether to only return the latest version of each process definition.
            When using this filter, pagination functionality is limited on clusters running on Elasticsearch or OpenSearch.
            The response contains no `startCursor` in the `page`, requests ignore the `from` and `before` in the `page`.
```